README.md

```diff
@@ -26,6 +26,11 @@ as well as The Diabetes Pedigree Function (which is a score that gives an idea a
 
 The final report can be found [here](https://github.com/UBC-MDS/diabetes_predictor_py/tree/main/analysis) or [this webpage](https://ubc-mds.github.io/diabetes_predictor_py/).
 
+## Dependencies
+- [Docker](https://www.docker.com/) 
+- [VS Code](https://code.visualstudio.com/download)
+- [VS Code Jupyter Extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
+
```

Comment:
I believe we still need `nb_conda_kernel`, `mamba`, `conda`, and `conda-lock`, please refer to our dependencies section from milestone 2 and add.

Reply:
These dependencies exist under  
## Developer Dependencies (starting line 106)

- conda (version 23.9.0 or higher)
- conda-lock (version 2.5.7 or higher)
- mamba (version 1.5.8 or higher)
- jupyterlab (version 4.0.0 or higher)
- nb_conda_kernels (version 2.3.1 or higher)
- Python and packages listed in [`environment.yml`](https://github.com/UBC-MDS/diabetes_predictor_py/blob/main/environment.yml)